example_cpp_smart_card_client_app/src/pp_module.cc

```diff
@@ -372,16 +404,19 @@ class PpInstance final : public pp::Instance {
   // This method is called by the constructor once all of the initialization
   // steps finish.
   void StartWorkInBackgroundThread() {
-    std::thread(&PpInstance::Work).detach();
+    std::thread(&PpInstance::Work, chrome_certificate_provider_api_bridge_)
+        .detach();
   }
 
   // This method is executed on a background thread after all of the
   // initialization steps finish.
-  static void Work() {
+  static void Work(
+      std::weak_ptr<ccp::ApiBridge> chrome_certificate_provider_api_bridge) {
     //
     // CHANGE HERE:
     // Place your custom initialization code here:
     //
+    ReportAvailableCertificates(chrome_certificate_provider_api_bridge);
```

Comment:
nit: Maybe let's add a hint that this call is still expected to be done in the real-world application. E.g.:

```
    //
    // CHANGE HERE:
    // Place your custom initialization code here:
    //

    // Report the currently available list of certificates immediately after the
    // initialization, according to requirements imposed by Chrome - see
    // <https://developer.chrome.com/extensions/certificateProvider#method-setCertificates>.
    ReportAvailableCertificates(chrome_certificate_provider_api_bridge);
```